test/OpenTelemetry.Exporter.Jaeger.Tests/Implementation/JaegerActivityConversionTest.cs

```diff
@@ -573,6 +574,11 @@ public void JaegerActivityConverterTest_Status_ErrorFlagTest(StatusCode expected
             return activity;
         }
 
+        private long TimeSpanToMicroseconds(TimeSpan timeSpan)
+        {
+            return timeSpan.Ticks / (TimeSpan.TicksPerMillisecond / 1000);
```

Comment:
Consider `timeSpan.Ticks / TimeSpan.TicksPerMillisecond * 1000`. (it's uncommon, but in case TicksPerMillisecond is a small number on new hardware).